lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2253,6 +2271,10 @@ static bool isExpressionEmittedInline(Operation *op) {
   if (op->hasOneUse() && isa<hw::OutputOp>(*op->getUsers().begin()))
     return true;
 
+  // Always emit constant file descriptors inline.
```

Comment:
But that isn't what this does, since I can use an actual constant to do the same thing and they have different code paths.